spec/helpers/deployment_helper_spec.rb

```diff
@@ -7,4 +7,21 @@
       expect(helper.deployment_count_options).to eq expected
     end
   end
+
+  describe '#provider_label' do
+
+    it 'returns the provider image when present' do
+      img_path = 'providers/logo_existing_image_small.png'
+      allow(File).to receive(:exist?)
+        .with(Rails.root.join("app", "assets", "images", img_path))
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.